sieve_of_eratosthenes/sieveOfEratosthenes.c

```diff
@@ -0,0 +1,34 @@
+#include <stdio.h>
+#include <string.h>
+#include <math.h>     // for using square root 
+#include <stdbool.h>  // to include bool datatype 
+
+void sieveOfEratosthenes(int n) {
+	bool primes[n+1];
+	int i, sqrtOfn = sqrt(n)+1, j;
+	
+	memset(primes, true, sizeof(primes));
```

Comment:
Actually I was asking for something else. I get it now, needed a revise of memory allotment in `C`. Thanks.